### DISCUSSION TITLE
HTTP header is larger than 10240bytes

### DISCUSSION THREAD

**Discussion by d-shannon** [Kyon147/laravel-shopify#51] [General]
Hey all,

Occasionaly I get a "HTTP header is larger than 10240bytes" error. Deleting cookies appears to resolve the issue.

Was wondering if anyone else has experienced this?

Cheers

**Comment by Kyon147** [comment]
I've not seen this before, strange though as v17 does not use cookies. What version are you on?

**Comment by d-shannon** [comment]
Yes it's weird - im on the latest version. Was reported by a few different users to me, but I have not been able to replicate myself. Possible something specific to my environment...
